docs/dev/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "fast-track-test"
```

Comment:
I was pretty disappointed with poetry. Could we just use setup tools like we do in python client?

Reply:
Honestly I am the opposite. I would rather work out the headaches of using poetry with the Python Client than what we currently do. Maybe we can discuss this further?